packages/slice-machine/lib/models/common/MockConfig.ts

```diff
@@ -23,6 +23,15 @@ export const SliceMockConfig = {
     sliceMockConfig: SliceMockConfig,
     variationId: string
   ) {
+    if (Array.isArray(sliceMockConfig)) {
```

Comment:
sliceMockConfig can be an array ???

Reply:
where this function is called with an array instead of an object ?

Reply:
It can be an array or an object, 
on the image field for one, probably else where to as in some mocks it
`"sliceName": { "default" : {...} }`
and in others it's
`"sliceName: [ {varition: "default" ... }]`

Reply:
.slicemachine/asset/ExtraSlice/mock.json
looks like this
...so I'm not sure what's going on with the mock-config file
```
[
  {
    "variation": "default",
    "name": "Default",
    "slice_type": "extra_slice",
    "items": [],
    "primary": {
      "title": [
        {
          "type": "heading1",
          "text": "Synergize wireless e-commerce",
          "spans": []
        }
      ],
      "description": [
        {
          "type": "paragraph",
          "text": "Non ut fugiat proident anim ut consequat consectetur eu est magna qui sit voluptate cupidatat minim. Velit exercitation consequat sit dolore tempor enim fugiat est in fugiat.",
          "spans": []
        }
      ],
      "keyText": "Some key txt",
      "image": {
        "dimensions": {
          "width": 900,
          "height": 500
        },
        "alt": "Placeholder image",
        "copyright": null,
        "url": "https://images.unsplash.com/photo-1521206698660-5e077ff6f9c8?w=900&h=500&fit=crop"
      }
    }
  }
]```